src/lr_apply_tmpl_16.rs

```diff
@@ -715,10 +715,10 @@ unsafe extern "C" fn lr_stripe(
                     & LR_HAVE_BOTTOM as libc::c_int as libc::c_uint,
         );
         lr_fn.expect("non-null function pointer")(
-            p,
+            p.cast(),
```

Comment:
We'll want to put this behind a typed interface eventually.  I think when we genericize over bitdepth is a good time to do that.